error.c

```diff
@@ -522,12 +522,9 @@ rb_warn_deprecated_to_remove(const char *removal, const char *fmt, const char *s
 {
     if (!deprecation_warning_enabled()) return;
 
-    va_list args;
-    va_start(args, suggest);
-    VALUE mesg = warning_string(0, fmt, args);
-    va_end(args);
-
-    warn_deprecated(mesg, removal, suggest);
+    with_warning_string(mesg, 0, fmt) {
```

Comment:
`fmt` is not the last named parameter in this function.

Reply:
@nobu 
Thanks for comments.
Fixed it(https://github.com/ruby/ruby/pull/6471/commits/74459a1f944b321fa942c982b85db6db65ea52b5)

Reply:
And `rb_warn_deprecated` functions change fail in CI that may be same problem.

Reply:
You can use `with_warning_string` now.

Reply:
I meant

```suggestion
    with_warning_string_from(mesg, 0, fmt, suggest) {
```

Reply:
@nobu 
Thanks.
Fixed it(`with_warning_string_from` macro) (https://github.com/ruby/ruby/pull/6471/commits/6ad3b7ef75158d7171579e430eb62b3b575fef24)